tests/test_validator_integration.py

```
@@ -0,0 +1,975 @@
+#  6.  Config fields in YAML but absent from train.py forwarding            #
+# ======================================================================== #
+
+
+class TestConfigFieldsReachTraining:
+    """Verify that every field defined in the shipped YAML configs is either
+    forwarded by train.py or explicitly documented as unused."""
+
+    KNOWN_NON_FORWARDED = {
+        "model", "data",
+        "training.prev_path", "training.freeze_components",
+    }
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)